internal/web3ext/web3ext.go

```diff
@@ -611,6 +611,18 @@ web3._extend({
 			params: 1,
 			inputFormatter: [web3._extend.formatters.inputTransactionFormatter]
 		}),
+		new web3._extend.Method({
+			name: 'getReceipts',
+			call: 'wemix_getReceipts',
```

Comment:
For clarity, I would recommend changing the method name "eth_getReceipts" to "eth_getReceiptsByBlockNumber".